src/plugins/aggregate-od/AggregateOd.vue

```diff
@@ -38,7 +38,7 @@
       label.checkbox(style="margin: 0 0.5rem 0 auto;")
           input(type="checkbox" v-model="showTimeRange")
           | &nbsp;{{ $t('duration') }}
-      time-slider.time-slider(v-if="headers.length > 0"
+      time-slider.time-slider(v-if="headers.length > 2"
```

Comment:
This v-if should be on line 36 instead. The entire "time" configuration should be hidden if there is only one column of data.

Reply:
Done :)